govcd/nsxt_alb_virtual_service_test.go

```diff
@@ -17,6 +17,13 @@ func (vcd *TestVCD) Test_AlbVirtualService(check *C) {
 	skipNoNsxtAlbConfiguration(vcd, check)
 	skipOpenApiEndpointTest(vcd, check, types.OpenApiPathVersion1_0_0+types.OpenApiEndpointAlbEdgeGateway)
 
+	failedAlbCleanup := true
+	defer func() {
+		if failedAlbCleanup {
+			vcd.removeLeftoverEntitiesByTestName(check, check.TestName())
+		}
+	}()
```

Comment:
By default a run of cleanup for this test is deferred, unless it is disabled at the end of the test